.github/workflows/deploy.yaml

```diff
@@ -1,20 +1,27 @@
-name: DEPLOY
+name: Deploy via ftp
 on:
   push:
-    branches:
-    - release
-
+    branches: [dev]
 jobs:
   deploy:
+    name: Deploy
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 14, 16, 18 ]
     steps:
-    - uses: actions/checkout@v2
-    - name: rsync deployments
-      uses: burnett01/rsync-deployments@5.2.1
+    - uses: actions/checkout@v3
+    - name: Faz build do projecto
+      uses: actions/setup-node@v3
       with:
-        switches: -avzr --delete
-        path: build/
-        remote_path: /var/www/html/
-        remote_host: ${{ secrets.RDC_IP }}
-        remote_user: ${{ secrets.RDC_SSH_USER }}
-        remote_key: ${{ secrets.RDC_SSH_PRIVATE_KEY }}
+        node-version: ${{ matrix.node }}
+      - run: npm run build
+    - name: Actualiza site via FTP
+      uses: sebastianpopp/ftp-action@releases/v2
+      with:
+        host: ${{ secrets.FTP_SERVER }}
```

Comment:
@isabelcosta @toothlesspanda 
In order to test this we need to add the secrets to Github's secret manager but I currently don't have the creds for the existing FTP user.

Reply:
This was shared IRL

Reply:
podemos também usar o comando já existente no projecto para fazer isto